### DISCUSSION TITLE
the reported minutes used don't reflect my repositories

### DISCUSSION THREAD

**Discussion by saasworks-cloud** [community/community#27658] [Actions]
hi,
i think there might be a bug.  i am told i have used 2,200 minutes within about 3 hours of opening 3 hours of opening the account.  at most, i run 3 versions of Node.js across 2 OS, and there are only 3 repositories running tests.  My two biggest actions billed a total of 3 hours time, and the average across the other 10 actions was about 10-20 minutes.  That means on the high side of estimation I hit only about 6 hours of testing.  On the lower side 4.5 hours.  clearly 6 hours of billable time is 360 minutes and not 2,200 minutes!!!

I noticed the MacOS jobs in my action were hanging today so I set a timeout of 5 minutes.  

something seems wrong or broken.  please help.

**Comment by airtower-luna** [comment]
That sounds like you should [contact support](https://support.github.com/request). Us other users can't check the runs in in your private repositories, let alone fix your billing. :wink:
